app/controller/settings.js

```diff
@@ -0,0 +1,33 @@
+var fs = require('fs');
+var express = require('express');
+
+var router = express.Router();
```

Comment:
Should I use an express function and assign it to a new variable? (var config = express(); )